### PR TITLE
Update enable-features-office-365.md

### PR DESCRIPTION
**Pull request by Exnav29**
Added the "Skype for Business with Teams collaboration" and "Skype for Business with Teams collaboration and meetings" coexistence modes which are available options in the Teams Admin Center.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/enable-features-office-365.md [hunk 1]
```diff
@@ -93,7 +93,15 @@ Switch this setting **On** to turn on scoped directory searches.
 You can use these settings to configure how your users will be upgraded from Skype for Business to Microsoft Teams. 
 
 #### Coexistence mode
-You can specify a coexistence mode: **Teams only**, **Islands** (Teams and Skype for Business will coexist), or **Skype for Business only**. The coexistence mode you choose determines the routing of incoming calls and chats and the app that is used by the user to initiate chats and calls or to schedule meetings. For more information about coexistence modes, go to [Understand Microsoft Teams and Skype for Business coexistence and interoperability](teams-and-skypeforbusiness-coexistence-and-interoperability.md).
+You can specify a coexistence mode: 
+
+- **Teams only**
+- **Islands** (Teams and Skype for Business will coexist)
+- **Skype for Business only**
+- **Skype for Business with Teams collaboration** (Users receive chats and calls and schedule meetings in Skype for Business but use  Teams for group collaboration)
```
Comment (JohanFreelancer9) [inline Teams/enable-features-office-365.md]:
```suggestion
- **Skype for Business with Teams collaboration** (users receive chats and calls and schedule meetings in Skype for Business but use  Teams for group collaboration)
```

File: Teams/enable-features-office-365.md [hunk 2]
```diff
@@ -93,7 +93,15 @@ Switch this setting **On** to turn on scoped directory searches.
...
+- **Skype for Business with Teams collaboration and meetings** (Users receive chats and calls in Skype for Business but use Teams for group collaboration and to schedule meetings)
```
Comment (JohanFreelancer9) [inline Teams/enable-features-office-365.md]:
```suggestion
- **Skype for Business with Teams collaboration and meetings** (users receive chats and calls in Skype for Business but use Teams for group collaboration and to schedule meetings)
```

File: Teams/enable-features-office-365.md [hunk 3]
```diff
@@ -93,7 +93,15 @@ Switch this setting **On** to turn on scoped directory searches.
...
+
+The coexistence mode you choose determines the routing of incoming calls and chats and the app that is used by the user to initiate chats and calls or to schedule meetings. For more information about coexistence modes, go to [Understand Microsoft Teams and Skype for Business coexistence and interoperability](teams-and-skypeforbusiness-coexistence-and-interoperability.md).
```
Comment (JohanFreelancer9) [inline Teams/enable-features-office-365.md]:
```suggestion
The coexistence mode that you choose determines the routing of incoming calls and chats and the app that is used by the user to initiate chats and calls or to schedule meetings. For more information about coexistence modes, go to [Understand Microsoft Teams and Skype for Business coexistence and interoperability](teams-and-skypeforbusiness-coexistence-and-interoperability.md).
```

**Comment by scanum** [comment]
@LolaJacobsen PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**